packages/sdk/src/endpoints/AttributesEndpoint.ts

```diff
@@ -1,20 +1,52 @@
 import {
     ConnectorAttribute,
     ConnectorAttributes,
+    ConnectorRequest,
     ConnectorResponse,
-    CreateAttributeRequest,
+    CreateAndShareRelationshipAttributeRequest,
+    CreateIdentityAttributeRequest,
     ExecuteIQLQueryRequest,
     ExecuteIdentityAttributeQueryRequest,
     ExecuteRelationshipAttributeQueryRequest,
     ExecuteThirdPartyRelationshipAttributeQueryRequest,
     GetAttributesRequest,
-    GetValidAttributesRequest
+    GetValidAttributesRequest,
+    NotifyPeerAboutIdentityAttributeSuccessionRequest,
+    ShareIdentityAttributeRequest,
+    SucceedAttributeRequest,
+    SucceedAttributeResponse,
+    SucceedIdentityAttributeRequest,
+    SucceedRelationshipAttributeAndNotifyPeerRequest
 } from "../types";
 import { Endpoint } from "./Endpoint";
 
 export class AttributesEndpoint extends Endpoint {
-    public async createAttribute(request: CreateAttributeRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
-        return await this.post("/api/v2/Attributes", request);
+    public async createIdentityAttribute(request: CreateIdentityAttributeRequest): Promise<ConnectorResponse<ConnectorAttribute>> {
+        return await this.post("/api/v2/Attributes/CreateIdentityAttribute", request);
+    }
+
+    public async createAndShareRelationshipAttribute(request: CreateAndShareRelationshipAttributeRequest): Promise<ConnectorResponse<ConnectorRequest>> {
+        return await this.post("/api/v2/Attributes/CreateAndShareRelationshipAttribute", request);
+    }
+
+    public async succeedAttribute(request: SucceedAttributeRequest): Promise<ConnectorResponse<SucceedAttributeResponse>> {
```

Comment:
I'm confused, we now mix two different options: I find it strange to add a new "convenience route" succeedAttribute, but remove the old route createAttribute in the SDK (but still allow it as a route in the Connector to be backwards-compatible).

I see two options:
a) We have no convenience routes in the SDK, but POST /Attributes stays as deprecated in the Connector. Then succeedAttribute should be removed from the SDK and from the Connector. But we loose the ability to test /Attributes.
b) We have convenience routes within the Connector and its SDK, both routes won't be deprecated, then createAttribute needs to stay in the SDK. Thus we do not need a major version inc and still be backwards compatible (and can test all Connector routes)

@jkoenig134 @slavistan Could we have a call tomorrow to clarify this?

I would vote for b).

Reply:
I agree, there's an inconsistency. I think it's better to present a unified interface to integrators, so the connector API and the SDK should match.

A third option is to not implement the `succeedAttribute` convenience route altogether. It's new and it was only created under the false assumption, that it already existed in the past.

Reply:
Would this be my option a)? :D

Reply:
Yepp, that's right. @jkoenig134 We've got a tie here. Which is it to be, a or b?

Reply:
as we have no other convenience routes at the moment I'd "vote" with a, but this isn't a democracy, we should discuss this more deeply :D

Reply:
I'll be waiting for a verdict.

Reply:
No, lets talk together like Thomas suggested!